.github/workflows/int_test.yml

```diff
@@ -92,6 +92,8 @@ jobs:
   setup:
     name: "Check Inputs"
     runs-on: ubuntu-22.04
+    # Run on dispatch, or ensure scheduled runs only on dev branch
+    if: github.event_name != 'schedule' || (github.repository == 'azure/azure-iot-ops-cli-extension' && github.ref == 'refs/heads/dev')
```

Comment:
is the value of github.repository always lowercase?

Reply:
From my testing it's not case-sensitive.

I'll merge and make sure it runs tonight (since this repo is technically `Azure`), otherwise I'll PR the one-char fix tomorrow 😄 
![image](https://github.com/user-attachments/assets/472a0d00-79df-40b5-9864-0f4ff433f88a)

![image](https://github.com/user-attachments/assets/276fe11a-2590-4dda-b595-e798d44381cc)